modules/acquia_cms_tour/src/Services/StarterKitService.php

```diff
@@ -91,6 +122,16 @@ public function enableModules(string $starter_kit, string $demo_question = NULL,
       ->save();
   }
 
+  /**
+   * Handler for enabling modules.
+   *
+   * @param string $module
+   *   Variable holding the module name.
+   */
+  public static function enableModule(string $module) {
```

Comment:
enableModules and enableModule isn't confusing at first go?
Can we rename this functions **enableSingleModule** or something different? @rajeshreeputra

Reply:
done